### ISSUE TITLE
about '645q9f3ozytrac'

### ISSUE THREAD

**Issue by Sarah50** [equation314/hitcon-ctf-2018-writeup#1]
why do you choose '645q9f3ozytrac', there are many cores, and not every core can come to a final solution

**Comment by equation314** [comment]
Yes, there may be multiple solutions, such as `hitcon{ms1n!gzob8leic}`. I guess other solutions can also pass.
